src/design-system/Input.tsx

```diff
@@ -35,3 +36,21 @@ export function useInputValue<T>(props: InputValueProps<T>) {
 }
 
 export type InputBaseProps<T> = InputLabelProps & InputValueProps<T>;
+
+export const StyledInput = styled.input`
+  background: #e0e4e4;
+  border-radius: 2px;
+  border: none;
+  box-shadow: none;
+  box-sizing: border-box;
+  color: #00413e;
+  flex: 1 1 auto;
+  font-family: "Rubik", sans-serif;
+  font-size: 16px;
```

Comment:
To reduce the number of lines here, consider using the `font` shorthand property.

You can do:

```css
  font: 16px/1.2 "Rubik", sans-serif;
```

This would replace both the two lines here and `line-height` below. The thing I like the most about this property is that in encourages you to set everything font-related one line. This makes it easier to reason about how a font would look, since it's all grouped in the same place, and makes it harder to accidentally omit font-related properties.

Let me know if you think this is helpful. If you don't think so, no need to use it!